manifests/package.pp

```diff
@@ -12,9 +12,18 @@
     }
   } else {
     package { $forge_server::package:
-      ensure   => present,
+      ensure   => $::forge_server::pkg_ensure,
       provider => $::forge_server::provider
     }
   }
 
+  # If we are ensuring the latest gem package, uninstall old versions
+  if $::forge_server::pkg_ensure == 'latest' {
```

Comment:
I 100% agree with this and can definitely make the change.

That said, there's no way that I can tell to clean up all but a specific version of a gem. For instance, if I tried to use 1.7.4 but needed to roll back to 1.7.3 and wanted to keep everything 'clean'. The only way to do that is to explicitly remove every other version installed.

What are your thoughts on this feature as a whole? Do you think it brings any value? Should we just leave it up to the end users to handle maintenance and just scrap the cleanup funcationality altogether?